compliance/queryresultio/src/test/java/org/eclipse/rdf4j/query/resultio/sparqljson/SPARQLJSONTupleBackgroundTest.java

```diff
@@ -82,48 +78,38 @@ public void testBindings1()
 		assertEquals(7, handler.getBindingSets().size());
 
 		// Results are ordered, so first should be book6
-		assertEquals("http://example.org/book/book6",
-				handler.getBindingSets().get(0).getValue("book").stringValue());
```

Comment:
I would prefer if it breaks on all the arguments to the method, rather than arbitrarily on one.

Eg. like this:

```
assertEquals(
    "http://example.org/book/book6",
    handler.getBindingSets().get(0).getValue("book").stringValue()
);
```

Reply:
Fwiw in this particular case I kind of agree with you, but the way it currently wraps method arguments is "where necessary", which is how the Eclipse conventions define it. I had a quick look to change this but modifying this would require tweaking several wrapping settings ( = deviating further from the sensible defaults) and even then I can't quite get it how you prefer.

So I think I'll leave this as-is and live with the compromise.

Reply:
ok